paging.json

```diff
@@ -36,6 +37,33 @@
           }
         }
       }
+    },
+    "/{nextLink}": {
```

Comment:
Here is the next operation defined. The path should only contain the `nextLink` path fragment (and whatever query params you want to format, if necessary).

Even though the property is called odata.nextLink, you should use `nextLink` because that is the name by which we pass the next link